rtv/content.py

```diff
@@ -154,7 +163,7 @@ def from_url(cls, reddit, url, loader, indent_size=2, max_indent_level=8):
 
         try:
             with loader():
-                submission = reddit.get_submission(url, comment_sort='hot')
+                submission = reddit.get_submission(url.replace("http:","https:"), comment_sort='hot')
```

Comment:
Why is this necessary? If the URL is provided as ``http:``, it would make sense to honor that and let Reddit redirect us to the right location.